docs/js/main.js

```diff
@@ -54,15 +62,28 @@ modalCloseButton.addEventListener("click", () => {
 });
 
 // Functions
+function validateSearch(recipeInput) {
+    const acceptedLetter = /[a-zA-Z]/;
+    if (recipeInput.length === 0) {
+       searchMessages.innerText = "You didn't input anything!";
+    } else if (!recipeInput.match(acceptedLetter)) {
```

Comment:
This line means "else if there isn't a single letter in recipeInput."

If recipeInput = '%98u3', the match will result in ['u'], then !match will be false and the search message won't change.

But
```js
const unacceptable = /[^a-zA-Z]/;
else if (recipeInput.match(unacceptable) {  }
```
works